### PR TITLE
Mention limitations of the request show redesign

### PR DESCRIPTION
**Pull request by dmarcoux**
Preview:

Message at the top of the page + Build Results:
preview-1

Mentioned Issues:
preview-2

### CODE REVIEW

File: src/api/app/views/webui/request/beta_show.html.haml [hunk 1]
```diff
@@ -6,6 +6,21 @@
   = render partial: 'actions'
   = render partial: 'add_reviewer_dialog'
 
+.alert.alert-info{ role: 'alert' }
+  As part of the
+  = succeed ',' do
```
Comment (dmarcoux) [inline src/api/app/views/webui/request/beta_show.html.haml:11]: This is needed to avoid `beta program ,`, so now we have `beta program,` as it should be.

**Comment by dmarcoux** [comment]
[Make the beta view the default for people in beta (M)](https://trello.com/c/tuF8tYhx/1943-make-the-beta-view-the-default-for-people-in-beta-m)

File: src/api/app/views/webui/request/beta_show_tabs/_build_results.html.haml [hunk 2]
```diff
@@ -1 +1,7 @@
-Build results
+%p
+  The build results aren't here yet. For now, you can see them on the request's
+  = link_to('source package', package_show_path(project, package))
+  page. We're working on having them here as soon as possible.
```
Comment (hennevogel) [inline src/api/app/views/webui/request/beta_show_tabs/_build_results.html.haml:4]: This is not the request's page. It's the source package.

I would also not make promises like `as soon as possible` :)

**Review by hennevogel — CHANGES_REQUESTED** [review]

File: src/api/app/views/webui/request/beta_show_tabs/_build_results.html.haml (hunk 2 above)
Comment (dmarcoux) [reply src/api/app/views/webui/request/beta_show_tabs/_build_results.html.haml]: > This is not the request's page. It's the source package.

The link is on `source package`, so this is already the case.

> I would also not make promises like as soon as possible :)

Good point, I pushed some changes.

**Review by hennevogel — APPROVED** [review]

**Pull request merged**